ui/src/views/Settings.vue

```diff
@@ -22,16 +22,283 @@
-              :invalid-message="error.testField"
-              ref="testField"
-            ></cv-text-input>
+              ref="hostname"
+              tooltipAlignment="center"
+              tooltipDirection="right"
+            >
+              <template slot="tooltip">
+                <div
+                  v-html="
+                    $t(
+                      'settings.hostname_must_be_relevant_for_user_authentication'
+                    )
+                  "
+                ></div>
+              </template>
+            </NsTextInput>
+            <NsComboBox
+              v-model.trim="ldap_domain"
+              :autoFilter="true"
+              :autoHighlight="true"
+              :title="$t('settings.ldap_domain')"
+              :label="$t('settings.choose_ldap_domain')"
+              :options="domains_list"
+              :acceptUserInput="false"
+              :showItemType="true"
+              :invalid-message="$t(error.ldap_domain)"
+              :disabled="loading.getConfiguration || loading.configureModule"
+              tooltipAlignment="start"
+              tooltipDirection="top"
+              ref="ldap_domain"
+            >
+              <template slot="tooltip">
+                {{ $t("settings.choose_the_ldap_domain_to_use") }}
+              </template>
+            </NsComboBox>
+            <cv-toggle
+              value="letsEncrypt"
+              :label="$t('settings.request_https_certificate')"
+              v-model="isLetsEncryptEnabled"
+              :disabled="loading.getConfiguration || loading.configureModule"
+              class="mg-bottom"
+            >
+              <template slot="text-left">{{
+                $t("settings.disabled")
+              }}</template>
+              <template slot="text-right">{{
+                $t("settings.enabled")
+              }}</template>
+            </cv-toggle>
+            <!-- advanced options -->
+            <cv-accordion ref="accordion" class="maxwidth mg-bottom">
+              <cv-accordion-item :open="toggleAccordion[0]">
+                <template slot="title">{{ $t("settings.advanced") }}</template>
+                <template slot="content">
+                  <cv-toggle
+                    value="webadmin"
+                    :label="$t('settings.webadmin_status')"
+                    v-model="webadmin"
+                    :disabled="
+                      loading.getConfiguration || loading.configureModule
+                    "
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <template v-if="webadmin">
+                    <template v-if="hostname">
```

Comment:
UI widgets that appears when enabling Web Admin toggle should be indented (use `mg-left` class)